package.json

```diff
@@ -38,9 +38,10 @@
     "test-api": "mocha -R dot --recursive tests/api",
     "test-disablehttp": "DISABLE_HTTP=true mocha -R dot --recursive tests/disableHttp",
     "test-enforced": "USE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken",
+    "test-cache-persp": "ENABLE_CACHE_PERSPECTIVE=true mocha -R dot --recursive tests/enableCache",
```

Comment:
@iamigo I tried as you suggested, but I have created a var cacheEnabled using feature toggle in api/v1/helpers/nouns/perspectives.js which is called when app server starts, and hence cacheEnabled calculates to false when checking for feature. This results in failed tests even if I use toggleOverride.